src/core/SampleBuffer.cpp

```diff
@@ -957,10 +962,15 @@ void SampleBuffer::visualize( QPainter & _p, const QRect & _dr,
 	const int last = focus_on_range ? _to_frame : m_frames;
 	for( int frame = first; frame < last; frame += fpp )
 	{
+		float fade = 1.0f;
+		if (m_fading)
+		{
+			fade = fadefunc(frame);
+		}
 		l[n] = QPointF( xb + ( (frame - first) * double( w ) / nb_frames ),
-			( yb - ( m_data[frame][0] * y_space * m_amplification ) ) );
+			( yb - ( m_data[frame][0] * y_space * m_amplification * fade) ) );
```

Comment:
This will conflict with changes from a recent merge (#5765) but it's a simple fix. Not sure why Github didn't accuse a conflict though.